sw/snRuntime/tests/zero_mem.c

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <snrt.h>
+
+#include "printf.h"
+
+int main() {
+    int errors = 0;
+
+    uint32_t n_inputs = 4;
+
+    // Get memory locations
+    uint32_t *zero_mem = (void *)(snrt_zero_memory().start);
+    uint32_t *buffer_tcdm = (void *)snrt_cluster_memory().start;
+    uint32_t *buffer_golden = (void *)(snrt_cluster_memory().start + 128);
+
+    // printf("Zero Memory at %p\n", zero_mem);
+    // printf("TCDM Memory at %p\n", buffer_tcdm);
+
+    ///////////////
+    // CORE READ //
+    ///////////////
+    if (snrt_is_compute_core()) {
```

Comment:
Nitpick, all compute cores in a cluster will execute this section. It should still function correctly. Could also be changed to `if (snrt_cluster_compute_core_idx() == 0)`